x-pack/plugins/canvas/canvas_plugin_src/functions/common/dropdownControl.ts

```diff
@@ -52,8 +52,10 @@ export function dropdownControl(): ExpressionFunctionDefinition<
     fn: (input, { valueColumn, filterColumn, filterGroup }) => {
       let choices = [];
 
-      if (input.rows[0][valueColumn]) {
-        choices = uniq(input.rows.map((row) => row[valueColumn])).sort();
+      const filteredRows = input.rows.filter((row) => row[valueColumn]);
```

Comment:
This will filter out any falsy values, including `0` or `false`, which probably should be included. 

I think a more appropriate solution would just be to make sure `input.rows` has length > 0 and using whatever values come from that.  Or maybe we explicitly filter out `null` from the list, but perhaps null should be a valid option as well.